projects/steering/src/main.c

```diff
@@ -0,0 +1,50 @@
+#include "adc.h"
+#include "adc_periodic_reader.h"
+#include "event_queue.h"
+#include "gpio_it.h"
+#include "gpio_mcu.h"
+#include "interrupt.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
+#include "steering_control_stalk.h"
+#include "steering_digital_input.h"
+#include "steering_events.h"
+#define STEERING_CAN_DEVICE_ID 0x1

```

Comment:
this should be included in `can_msg_defs.h`, include that and use the value `SYSTEM_CAN_DEVICE_STEERING`. You'll probably also have to add codegen-tooling to the makefile, check the mci project makefile for details I think. Sorry, I think I missed this on first review.